### PR TITLE
There was an empty body test method

### PR DESCRIPTION
**Pull request by gambe94**
captureWithName() in RealWorldUnitTest.class was empty so I wrote a comment why is it empty.

### CODE REVIEW

**Comment by soosroland** [comment]
I also think it is good in that way.

File: pom.xml [hunk 1]
```diff
@@ -80,6 +80,21 @@
...
+            	<version>2.17</version>
+		<executions>
+                    <execution>
+                        <goals>
+                            <goal>checkstyle</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            	<configuration>
+                	<configLocation>/home/csabi/workspace/integell/JavaVerbalExpressions/checkstyle-checker.xml</configLocation>
```
Comment (lanwen) [inline pom.xml]: looks like its only yours path :)

File: src/test/java/ru/lanwen/verbalregextest/BasicFunctionalityUnitTest.java [hunk 2]
```diff
@@ -1,7 +1,9 @@
-package ru.lanwen.verbalregex;
+package ru.lanwen.verbalregextest;
```
Comment (lanwen) [inline src/test/java/ru/lanwen/verbalregextest/BasicFunctionalityUnitTest.java]: why?

**Comment by lanwen** [comment]
Please give a signal when it will be ready to review and merge! :)

**Pull request closed without merging**